vpr/SRC/route/route_timing.c

```diff
@@ -17,14 +19,26 @@
 #include "net_delay.h"
 #include "stats.h"
 #include "ReadOptions.h"
-
-
+ 
 using namespace std;
 
+/***************** Iterative connection based rerouting **********************/
 
+// array indexed by inet of lookup tables mapping terminal rr_nodes to its pin on the net
+// the reverse lookup of net_rr_terminals
+static vector<unordered_map<int,int>> rr_node_to_pin;

```

Comment:
It would be good to give an example.  As I under stand it you would use it as:

    //Get the net pin index of rr_node 'rr_node_index' on net 'net_index'
    int net_pin_index = rr_node_to_pin[net_index][rr_node_index]

We need to be careful with this since it could cause 'rr_node_index' to be inserted into the unordered_map if it was not included.
It might be good to wrap this in a function/class which would handle this case.